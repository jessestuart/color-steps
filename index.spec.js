const colorSteps = require('./index.js');

describe('color-steps utility', () => {
  it('should match base calucation with white and black', () => {
    expect(colorSteps('black', 'white')).toMatchSnapshot();
  });
});
