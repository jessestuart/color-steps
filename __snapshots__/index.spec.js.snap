// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color-steps utility should match base calucation with white and black 1`] = `
Array [
  "rgb(106,106,106)",
  "rgb(146,146,146)",
  "rgb(175,175,175)",
  "rgb(199,199,199)",
  "rgb(220,220,220)",
  "rgb(238,238,238)",
]
`;

exports[`color-steps utility should support alpha 1`] = `
Array [
  "rgba(0,0,0,0.5)",
]
`;

exports[`color-steps utility should support going from light to dark 1`] = `
Array [
  "rgb(238,238,238)",
  "rgb(220,220,220)",
  "rgb(199,199,199)",
  "rgb(175,175,175)",
  "rgb(146,146,146)",
  "rgb(106,106,106)",
]
`;
